helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/WagedRebalancer.java

```diff
@@ -109,7 +118,8 @@ private static AssignmentMetadataStore constructAssignmentStore(String metadataS
   }
 
   public WagedRebalancer(HelixManager helixManager,
-      Map<ClusterConfig.GlobalRebalancePreferenceKey, Integer> preference) {
+      Map<ClusterConfig.GlobalRebalancePreferenceKey, Integer> preference,
+      boolean isAsyncGlobalRebalance) {
```

Comment:
At least we need to keep the name consistent, sometimes it's global, sometimes it's baseline

Reply:
We have spent lots of effort on sync the names. Note that the process is called global rebalance. And the result is called baseline assignment. Here, the name is accurate.
If you feel the name does not match what I noted here, please comment. And I will fix it.

Reply:
On that saying, let me change the field name to be asyncGlobalRebalance. To avoid saying the same thing in two ways.